drivers/gpu/drm/tests/drm_framebuffer_test.c

```diff
@@ -332,29 +340,47 @@ static struct drm_mode_config_funcs mock_config_funcs = {
 
 static int drm_framebuffer_test_init(struct kunit *test)
 {
-	struct drm_device *mock;
+	struct drm_device_mock *mock;
+	struct drm_device *drm;
```

Comment:
There are some occurrences of *drm, but it's way more common to call it *dev

Reply:
Okay, I'll rework my commits to replace it to *dev.

Thanks!

Reply:
Done!